sdk/digitaltwins/digitaltwins/src/digitalTwinsClient.ts

```diff
@@ -0,0 +1,851 @@
+   */
+  public decomissionModel(modelId: string, updateModel: any[]): Promise<RestResponse> {
+    return this.client.digitalTwinModels.update(modelId, updateModel);
+  }
+
+  /**
+   * Delete a model.
+   *
+   * @param modelId The Id of the model to delete.
+   * @returns The http response.
+   */
+  public deleteModel(modelId: string): Promise<RestResponse> {
+    return this.client.digitalTwins.deleteMethod(modelId);
+  }
+
+  /**
+   * Get an event route.
+   *
+   * @param modelId The Id of the event route.
+   * @returns The application/json event route and the http response.
+   */
+  public getEventRoute(eventRouteId: string): Promise<EventRoutesGetByIdResponse> {
+    return this.client.eventRoutes.getById(eventRouteId);
+  }
+
+  /**
+   * @internal
+   * @ignore
+   * Deals with the pagination of {@link list}.
+   *
+   * @param {PageSettings} continuationState An object that indicates the position of the paginated request.
+   * @param {EventRoutesListOptionalParams} [options] Common options for the iterative endpoints.
+   *
+   */
+  private async *getEventRoutesPage(
+    options: EventRoutesListOptionalParams,
+    continuationState: PageSettings
+  ): AsyncIterableIterator<EventRoutesListNextResponse> {
+    if (continuationState.continuationToken == null) {
+      const optionsComplete: EventRoutesListOptionalParams = {
+        maxresults: continuationState.maxPageSize,
+        ...options
+      };
+      const listResponse = await this.client.eventRoutes.list(optionsComplete);
+      continuationState.continuationToken = listResponse.nextLink;
+      yield listResponse;
+    }
+    while (continuationState.continuationToken) {
+      const listResponse = await this.client.eventRoutes.listNext(
+        continuationState.continuationToken,
+        options
+      );
+
+      continuationState.continuationToken = listResponse.nextLink;
+      yield listResponse;
+    }
+  }
+
+  /**
+   * @internal
+   * @ignore
+   * Deals with the iteration of all the available results of {@link list}.
+   * @param {EventRoutesListOptionalParams} [options] Common options for the iterative endpoints.
+   */
+  private async *getEventRoutesAll(
+    options: EventRoutesListOptionalParams
+  ): AsyncIterableIterator<EventRoute> {
+    const f = {};
+
+    for await (const page of this.getEventRoutesPage(options, f)) {
+      for (const item of page) {
+        yield item;
+      }
+    }
+  }
+
+  /**
+   * List the event routes in a digital twins instance.
+   *
+   * @param maxItemCount The maximum number of items to retrieve per request. The server may choose to return less than
+   * the requested max. Default value: -1.
+   * @returns The application/json event route and the http response.
+   */
+  public listEventRoutes(
+    maxItemCount: number = -1
+  ): PagedAsyncIterableIterator<EventRoute, EventRoutesListNextResponse> {
+    var options: EventRoutesListOptionalParams & PageSettings = {};
+    if (maxItemCount != -1) {
```

Comment:
lint: should use strict equality checks

```suggestion
    if (maxItemCount !== -1) {
```

Reply:
If the linter isn't running, you may have to tweak .eslintrc.json

Reply:
Fixed

---
In reply to: [447263018](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r447263018) [](ancestors = 447263018)

Reply:
Fixed

---
In reply to: [447262868](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r447262868) [](ancestors = 447262868)